src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -371,6 +371,14 @@ public enum EvalMetricType
         [TlcModule.SweepableDiscreteParam("CatL2", new object[] { 0.1, 0.5, 1, 5, 10 })]
         public double CatL2 = 10;
 
+        [Argument(ArgumentType.Multiple,
```

Comment:
>Argument [](start = 9, length = 8)

please add ShortName, if someone would ever need to use that in command line, he would curse you for typing this words again and again.